dts/x86/ia32.dtsi

```diff
@@ -0,0 +1,51 @@
+#include "skeleton.dtsi"
+#include "mem.h"
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "intel,ia32";
+			reg = <0>;
+		};
+
+	};
+
+	flash0: flash@00001000 {
+		reg = <0x00001000 DT_FLASH_SIZE>;
+	};
+
+	sram0: memory@00400000 {
+		device_type = "memory";
+		compatible = "mmio-sram";
+		reg = <0x00400000 DT_SRAM_SIZE>;
+	};
+
+
+	soc {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		compatible = "simple-bus";
+		ranges;
+
+
+		uart0: uart@f0008000 {
+			compatible = "intel,ia32-uart";
```

Comment:
this should use the 16550 see ./dts/arm/yaml/ns16550.yaml (we need to move that)

Reply:
dts/arm/yaml/ns16550.yaml exists only in the ARM branch?

Reply:
@galak  i don't see any reference to ns16550.yam in yaml folder of arm